CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [11.0.0](https://github.com/theforeman/puppet-foreman_proxy_content/tree/11.0.0) (2020-05-15)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy_content/compare/9.1.1...11.0.0)
+
+**Breaking changes:**
+
+- Use modern facts [\#258](https://github.com/theforeman/puppet-foreman_proxy_content/issues/258)
+
+**Implemented enhancements:**
+
+- Fixes [\#29214](https://projects.theforeman.org/issues/29214): Add EL8 support to metadata [\#255](https://github.com/theforeman/puppet-foreman_proxy_content/pull/255) ([ehelms](https://github.com/ehelms))
+- Refs [\#29211](https://projects.theforeman.org/issues/29211): Use extlib 5.x for EL8 support [\#254](https://github.com/theforeman/puppet-foreman_proxy_content/pull/254) ([ehelms](https://github.com/ehelms))
+- Refs [\#29214](https://projects.theforeman.org/issues/29214): Only install Pulp and Qpid on El7 [\#251](https://github.com/theforeman/puppet-foreman_proxy_content/pull/251) ([ehelms](https://github.com/ehelms))
+- Fixes [\#29278](https://projects.theforeman.org/issues/29278): Use default CA for crane SSL chain [\#245](https://github.com/theforeman/puppet-foreman_proxy_content/pull/245) ([ehelms](https://github.com/ehelms))
+- Fixes [\#28901](https://projects.theforeman.org/issues/28901) - Support SSL connection for external Pulpcore PostgreSQL… [\#244](https://github.com/theforeman/puppet-foreman_proxy_content/pull/244) ([wbclark](https://github.com/wbclark))
+- Use loose coupling to Foreman's Apache config [\#243](https://github.com/theforeman/puppet-foreman_proxy_content/pull/243) ([ekohl](https://github.com/ekohl))
+
+**Fixed bugs:**
+
+- Fixes [\#29589](https://projects.theforeman.org/issues/29589) - Ensure pulpcore before proxy [\#252](https://github.com/theforeman/puppet-foreman_proxy_content/pull/252) ([ekohl](https://github.com/ekohl))
+
+**Merged pull requests:**
+
+- Refs [\#28901](https://projects.theforeman.org/issues/28901) - support external postgres database in pulpcore [\#241](https://github.com/theforeman/puppet-foreman_proxy_content/pull/241) ([wbclark](https://github.com/wbclark))
+
+## [9.1.1](https://github.com/theforeman/puppet-foreman_proxy_content/tree/9.1.1) (2020-03-11)
```

Comment:
You should move this between 9.1.0 and 10.0.0

Reply:
chronologically it was released after 10.0.0 but I can move it to respect the version order instead

Reply:
Hmm, that's a good point. Normally I don't include those releases anymore because it gets confusing. If you read something was fixed in 9.1.1 but then it's not in 10.0.0 you might be surprised. Maintaining branches is ... complicated.